docs/make.jl

```diff
@@ -140,8 +142,8 @@ makedocs(bib,
     format = format,
      pages = pages,
    modules = [Oceananigans],
-   doctest = true,
-    strict = true,
+   doctest = false,
```

Comment:
Why this became `false`?

Reply:
It shouldn't be, I agree.  When I tried making the docs with both doctest and strict to be true, it failed, hence #1455 .  I agree these should be turned back on and maybe now they wil compile, I don't know.

Reply:
As discussed in #1455, if you delete all `.nc` files and `.jld2` files before running the docs they should be fine.

Reply:
They are true now